syntax = "proto3";
package file_trans;

service WebpackServer {
  rpc SendFile(Request) returns (stream FileChunk);
}

message Request {
  string filename = 1;
}

message FileChunk {
  bytes chunk = 1;
}

